### ISSUE TITLE
[PREDICTION, VIZ] Discussion about spatial models 

### ISSUE THREAD

**Issue by eduardocorrearaujo** [Mosqlimate-project/Data-platform#144]
I was wondering how to integrate spatial models into our prediction workflow, and I have some questions to discuss. 

First: Is our aim here to show the performance predicting some target (test sample) or the performance over training data? 

If we want to show the performance over some target, we must assume that each prediction sent refers to the same time interval. For example, if I have a model that predicts the number of cases in the next year and I want to send the predictions in 2022 and 2023, I need to send it as different prediction elements. Otherwise, how will we create a map plot? 

Second: How will we define the risk models? For example, if I have a binary classifier model that predicts whether next year will be an epidemic, is it interesting to put it on the platform? In this case, to compare to other models, we need to define what is an epidemic. There is a criterion for more than 300 cases by 100k. Will we use it?

This would be an output example of this class of model:
 
![Captura de Tela 2023-12-18 às 17 29 31](https://github.com/Mosqlimate-project/Data-platform/assets/65051489/dd79cb08-89dd-4d5b-9cfa-8fabb1ecbcb9)

Since we are looking for just one-time interval, we can not look at metrics as false negatives and accuracy.

**Comment by fccoelho** [comment]
Yes maps must have a timestamp: It can correspond to a representation of a moment, or the accumulated effects over a period of time, but in every case it must have a date of reference, and then the platform will automatically select comparable maps based on this and other characteristics as it already does with the temporal predictions.

**Comment by fccoelho** [comment]
The definition of risk used on a model must be documented in each model, there is no way to force everyone to use the same definition. Having said that, we can set up competitions  in which  we ask people to predict a specific target, be it risk or some other variable.

**Comment by eduardocorrearaujo** [comment]
@fccoelho can we close this issue or add a low-prio?

**Comment by fccoelho** [comment]
Marca como low-prio
